phac-nml/irida-next
# Localize member & group link mailers

## What does this PR do and why?
Sending member & group link emails to users in the locale set within their account.
A continuation of #571. Fixes #566.

## Screenshots or screen recordings
Please see #542 & #564.

## How to set up and validate locally
Please see #542 & #564.

Verify users & managers received their email in their locale of choice.

Try changing the locale drop-down in the [member](http://localhost:3000/rails/mailers/member_mailer) & [group_link](http://localhost:3000/rails/mailers/group_link_mailer) mailer previews.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
